pkg/env/var_test.go

```diff
@@ -43,6 +43,11 @@ func TestString(t *testing.T) {
 		t.Errorf("Expected not present")
 	}
 
+	v = ev.Get()
+	if v != "123" {
+		t.Errorf("Expected 123, got %s", v)
```

Comment:
do these error message pass the linter? I thought they had to start with lowercase?

Reply:
linter doesn't care about the testing.T calls.